contrib/slynk-mrepl.lisp

```diff
@@ -97,7 +97,7 @@ Set this to NIL to turn this feature off.")
   (declare (ignore subchar arg))
   (let* ((*readtable*
            (let ((table (copy-readtable nil)))
-             (set-macro-character #\: (lambda (&rest args) nil) nil table)
+             (set-macro-character #\: (constantly nil) nil table)
```

Comment:
This is nice, but what does it have to do with #134?

Reply:
Yeah, unrelated